src/lib/jsonStatham.js

```diff
@@ -49,7 +49,12 @@ var jsonStatham = {
         if (error.responseJSON)
           errorJSON = error.responseJSON
         else if (error.responseText)
-          errorJSON = JSON.parse(error.responseText)
+          try {
+            errorJSON = JSON.parse(error.responseText)
+          } catch(e) {
```

Comment:
@BJK CodeClimate is right: space after `catch` and you should use `parseError` or something instead of `e`--actually you don't really need any parameter here since you don't actually use it anywhere